src/Microsoft.Data/DataFrame.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Data
+{
+    /// <summary>
+    /// A DataFrame 
+    /// </summary>
+    public partial class DataFrame
+    {
+        internal DataFrameTable _table;
+        public DataFrame()
+        {
+            _table = new DataFrameTable();
+        }
+
+        public long RowCount => _table.RowCount;
+        public int ColumnCount => _table.ColumnCount;
+        public IList<string> Columns()
+        {
+            var ret = new List<string>();
+            for (int ii = 0; ii < ColumnCount; ii++)
```

Comment:
Any particular reason you use `ii` -- is this standard in this domain? We usually use `i`, `j` .. or something descriptive

Reply:
Nah. I'll change that now. I started doing that a while back to make the loop variable more unique so that I didn't get as many hits when I used "Find"